amd/src/state_change.js

```diff
@@ -84,6 +84,10 @@ define(['jquery', 'core/ajax', 'core/notification'], function($, Ajax, Notificat
                     Notification.alert(results.status, results.message);
                     pendingPromise.resolve();
                     lastSelectedState = stateChangeSelect.val();
+                    const currentSelectedText = stateChangeSelect.find('option:selected').text();
+                    Str.get_string('changecurrentstate', 'studentquiz', currentSelectedText).done(newString => {
+                        currentStateText.html(newString);
+                    });
                     // Reload the Studentquiz page.
                     window.opener.location.reload();
```

Comment:
If we are doing window.opener.location.reload, why have to code above to update the page first?

Reply:
Hi @timhunt,

the script above reloads the student quiz bank view, and does not reload the preview page.

Reply:
Ah! I get it now. Thank you for explaining :-)